schematools/contrib/django/factories.py

```diff
@@ -121,7 +130,7 @@ def handle_relation(
             # In schema foreign keys should be specified without _id,
             # but the db_column should be with _id
             kwargs["db_column"] = f"{to_snake_case(field.name)}_id"
-            kwargs["db_constraint"] = False  # relation is not mandatory
+            # kwargs["db_constraint"] = False  # relation is not mandatory
```

Comment:
`db_constraint = False` was nodig voor de datasets met niet-bestaande relaties.

Default is `True` en dus kan probleemen veroorzaken met imports.